CONTRIBUTING.md

```diff
@@ -92,17 +89,37 @@ it will create a new folder inside `packages` with a simple boilerplate for your
 
 ## Publishing
 
-Install lerna:
-
-```bash
-$ npm install -g lerna@2.0.0-beta.34
-```
+### Prerelease
```

Comment:
This is just replacing the old publishing stuff in CONTRIBUTING with the content from the now-deleted RELEASING